website/templates/issue.html

```diff
@@ -5,7 +5,6 @@
 {% providers_media_js %}
 {% block title %}Bug on {{ object.domain_title }} - {{ object.description }}{% endblock %}
 {% block og_title %}Bug on {{ object.domain_title }}{% endblock %}
-{% block og_image %}{{ object.screenshot.url }}{% endblock %}
```

Comment:
please put the original screenshot here and if there are multiple images, put the first image